src/page-modules/assistant/details/index.tsx

```diff
@@ -37,7 +43,46 @@ export function AssistantDetails({ tripPattern }: AssistantDetailsProps) {
               )
             : t(PageText.Assistant.details.header.title)}
         </Typo.h2>
+        <div className={style.tripDetails}>
+          <div className={style.date}>
+            <MonoIcon icon="time/Date" />
+            <Typo.p textType="body__primary">{weekdayAndDate}</Typo.p>
+          </div>
+          <div className={style.duration}>
+            <MonoIcon icon="time/Duration" />
+            <Typo.p textType="body__primary">{timeRange}</Typo.p>
+            <Typo.p textType="body__primary--bold">
+              {t(PageText.Assistant.details.header.travelTime(tripDuration))}
+            </Typo.p>
+          </div>
+        </div>
```

Comment:
Could this instead of being custom hook, be a separate component?

Reply:
Now were essentially decoupling data transformation and UI. But they are semantically tightly coupled. Maybe introducing this as a component and instead of having `useTripDetailsHeader` let this be transformed inside the component, it is coupled more correctly?

Reply:
I think that makes sense 👍 I've updated to reflect this now